08-projects.Rmd

```diff
@@ -4366,7 +4370,7 @@ sqlite3 /class/datamine/data/lahman/lahman.db
 - The first 10 results of the query.
 ```
 
-##### 7. Find the `playerID` for Bob Lemon. What year and team was he on when he pitched the most wins (use table `pitching`)? What year and team did he win the most games as a manager (use table `managers`)?
+##### 7. Find the `playerID` for Bob Lemon. What year and team was he on when he got the most wins as a pitcher (use table `pitching`)? What year and team did he win the most games as a manager (use table `managers`)?
```

Comment:
As someone who know nothing about baseball, I would have thought "pitched the most wins" meant "had the most successful plays when pitching, regardless of the game results". Hence, I added the clarification.